scripts/dashboard/build-dashboard.js

```diff
@@ -30,6 +32,21 @@ async function getHotDiscussions(discussions) {
             0
           )
           : interactionsCount;
+
+        console.log("Processed discussion:", {
+          id: discussion.id,
+          isPR,
+          isAssigned: !!discussion.assignees.totalCount,
+          title: discussion.title,
+          author: discussion.author ? discussion.author.login : '',
+          resourcePath: discussion.resourcePath,
+          repo: 'asyncapi/' + discussion.repository.name,
+          labels: discussion.labels ? discussion.labels.nodes : [],
+          score:
+            finalInteractionsCount /
+            Math.pow(monthsSince(discussion.timelineItems.updatedAt) + 2, 1.8),
+        });
```

Comment:
That's code duplication, assign this object to a variable, then return and log that variable.